homeassistant/components/lcn/config_flow.py

```diff
@@ -0,0 +1,30 @@
+"""Config flow to configure the LCN integration."""
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_IP_ADDRESS, CONF_PORT
+
+from .const import DOMAIN
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class LcnFlowHandler(config_entries.ConfigFlow):
+    """Handle a LCN config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_PUSH
+
+    async def async_step_import(self, data):
+        """Import existing configuration from LCN."""
+        # check if we already have a host with the same name configured
+        host_name = data.pop(CONF_HOST)
+        unique_id = f"{data[CONF_IP_ADDRESS]}:{data[CONF_PORT]}"
```

Comment:
How did you decide between using `host` as a `unique_id` vs using the IP + Port? It seems like the host is also used as a the canonical id (referenced by the entities in config).  No problem with this, but i thought it might help me understand better.

Reply:
Basically I need an identifier of the LCN hub (which sticks to one config_entry) which is also part of the entities `unique_id`.
For this I could use the `config_entry.entry_id` or the `config_entry.unique_id`. In the end this doesn't make any difference.
But I only found code which uses the `entry_id` as part of the entities unique_id. So I decided to use that...

Reply:
We don't allow ip address as unique id as ip addresses may change. In the config flow we can just check the existing entries' data for duplicate ip address to guard for duplicates. We won't use the unique id api then, unless there's some other good identifier we can use.

For entities it's ok to use the config entry entry_id as unique_id.

Reply:
Thanks for clarification. I replaced the unique_id checking with a check against the entries' data.
For the entities' I stick to the entry_id as a part of  the unique_id.